functions/assignOrder/config.json

```diff
@@ -0,0 +1,5 @@
+{
+    "name": "assignOrder",
+    "private": false,
+    "run_as_system": true
```

Comment:
for the functions which need to use the context to get user data I think it needs to not run as system